samples/get_alert_suppression_state.py

```diff
@@ -0,0 +1,14 @@
+import orionsdk
+
+server = 'orion_server'
+username = 'username'
+password = 'password'
+ip_address = '192.168.1.1'  # change this to the node you want to check
+
+
+sw = orionsdk.SwisClient(server, username, password)
+uri = sw.query("SELECT Uri FROM Orion.Nodes WHERE NodeID=@node_id", IPAddress=ip_address)['results'][0]['Uri']
```

Comment:
It looks like this script needs to be updated.  The `WHERE` clause isn't quite right.  It should be `WHERE IPAddress=@IPAddress` instead of looking for a node ID that isn't being supplied.

Reply:
You are correct I have since updated the queries in each of the 3 files I added to reflect the correct IPAddress=@IPAddress